purchase_order_general_discount/models/res_company.py

```diff
@@ -0,0 +1,32 @@
+# Copyright 2019 Tecnativa - David Vidal
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+from odoo import _, api, fields, models
+
+
+class ResCompany(models.Model):
+    _inherit = "res.company"
+
+    purchase_general_discount_field = fields.Selection(
+        selection="_get_purchase_discount_fields",
+        string="Purchase Discount Field",
+        default="discount",
+        help="Set the purchase line discount field in wich the "
```

Comment:
replace wich for which

Reply:
I have replaced all occurrences of `wich`